src/languages/en.ts

```diff
@@ -1537,6 +1537,12 @@ export default {
             invitePeople: 'Invite new members',
             genericFailureMessage: 'An error occurred inviting the user to the workspace, please try again.',
             pleaseEnterValidLogin: `Please ensure the email or phone number is valid (e.g. ${CONST.EXAMPLE_PHONE_NUMBER}).`,
+            user: 'user',
+            users: 'users',
+            invited: 'invited',
+            removed: 'removed',
```

Comment:
Can you observe this message when a user is removed from a room?

Reply:
We should add a comment about it, at least, maybe in the function which builds elements